host restart-all コマンドでエラーが発生する

BRHDLの画面上で起動すると問題なく開始する
v0.5.1

```
ubuntu@resonite-headless:~/reso-headless$ ./brhcli host list
ID: J2QUJB11kohwEnUN, Name: TriVR, Status: EXITED
ubuntu@resonite-headless:~/reso-headless$ ./brhcli host restart-all 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x673fb4]

goroutine 17 [running]:
github.com/hantabaru1014/baru-reso-headless-controller/usecase.(*HeadlessHostUsecase).resolveTagToUse(0x40001bf0f0?, {0xb578c8?, 0x1233f60?}, 0x0)
        /home/runner/work/baru-reso-headless-controller/baru-reso-headless-controller/usecase/headless_host_usecase.go:163 +0xf4
github.com/hantabaru1014/baru-reso-headless-controller/usecase.(*HeadlessHostUsecase).HeadlessHostRestart(0x4000241a40, {0xb578c8, 0x1233f60}, {0x400022d470?, 0x4000048d00?}, 0x0, 0x0?, 0x258)
        /home/runner/work/baru-reso-headless-controller/baru-reso-headless-controller/usecase/headless_host_usecase.go:90 +0x90
github.com/hantabaru1014/baru-reso-headless-controller/cmd/cli/commands.NewHostCommand.func2.1(0x40004100f0)
        /home/runner/work/baru-reso-headless-controller/baru-reso-headless-controller/cmd/cli/commands/host.go:70 +0x7c
created by github.com/hantabaru1014/baru-reso-headless-controller/cmd/cli/commands.NewHostCommand.func2 in goroutine 1
        /home/runner/work/baru-reso-headless-controller/baru-reso-headless-controller/cmd/cli/commands/host.go:68 +0x128
```

--- a/usecase/headless_host_usecase.go
+++ b/usecase/headless_host_usecase.go
@@ -160,7 +160,7 @@ func (hhuc *HeadlessHostUsecase) resolveTagToUse(ctx context.Context, tagInput *
 		if len(tags) == 0 {
 			return "", errors.New("no available container image tags")
 		}
-		wantPreRelease := *tagInput == "latestPreRelease"
+		wantPreRelease := tagInput != nil && *tagInput == "latestPreRelease"
 		for _, tag := range slices.Backward(tags) {
 			if tag.IsPreRelease == wantPreRelease {
 				return tag.Tag, nil

